test/os/file/mime/text.test.js

```diff
@@ -3,16 +3,6 @@ goog.require('os.file.mime.mock');
 goog.require('os.file.mime.text');
 
 describe('os.file.mime.text', function() {
```

Comment:
Is this no longer a valid testing concern? If it doesn't work on "first N characters" anymore, can't you instead feed the file to this test?

Reply:
This test was removed since chardetng won't return a false value to satisfy the test. Also I'm not sure I follow what your suggesting here..

Reply:
My thought here was that there could be another way to test the same thing as the old test. If not, no big deal.